db/migrate/20170801184720_create_versions.rb

```diff
@@ -1,80 +1,21 @@
 # This migration creates the `versions` table, the only schema PT requires.
 # All other migrations PT provides are optional.
 class CreateVersions < ActiveRecord::Migration[4.2]
-  # Class names of MySQL adapters.
-  # - `MysqlAdapter` - Used by gems: `mysql`, `activerecord-jdbcmysql-adapter`.
-  # - `Mysql2Adapter` - Used by `mysql2` gem.
-  MYSQL_ADAPTERS = [
-    "ActiveRecord::ConnectionAdapters::MysqlAdapter",
-    "ActiveRecord::ConnectionAdapters::Mysql2Adapter"
-  ].freeze
-
   # The largest text column available in all supported RDBMS is
   # 1024^3 - 1 bytes, roughly one gibibyte.  We specify a size
   # so that MySQL will use `longtext` instead of `text`.  Otherwise,
   # when serializing very large objects, `text` might not be big enough.
   TEXT_BYTES = 1_073_741_823
 
   def change
-    create_table :versions, versions_table_options do |t|
-      t.string   :item_type, item_type_options
+    create_table :versions do |t|
+      t.string   :item_type, null: false
       t.integer  :item_id,   null: false
       t.string   :event,     null: false
       t.string   :whodunnit
       t.text     :object, limit: TEXT_BYTES
-
```

Comment:
If we held on to this comment block, I think this migration would match the currently generated upstream one:

https://github.com/paper-trail-gem/paper_trail/blob/master/lib/generators/paper_trail/install/templates/create_versions.rb.erb

Reply:
Comment restored :-)